libs/antsi/.gitignore

```diff
@@ -0,0 +1,13 @@
+### Rust template
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
```

Comment:
I have never seen a `debug` folder and `target` is captured in the top level

Reply:
This is taken directly from the deer gitignore. This is generated by `Cargo`.